sktime/forecasting/ets.py

```diff
@@ -238,7 +239,29 @@ def __init__(
         self.n_jobs = n_jobs
 
         super(AutoETS, self).__init__(random_state=random_state)
-
+        
+        if self.auto:
+            # If auto=True, check if trend, damped_trend, seasonal, or error have been passed in by user
+            if any(param is not None for param in [trend, damped_trend, seasonal, error]):
+                warnings.warn(
+                    "The user-specified parameters provided alongside auto=True in AutoETS may not be respected. "
+                    "The AutoETS function automatically selects the best model based on the information criterion, ignoring "
+                    "the error, trend, seasonal, and damped_trend parameters when auto=True is set. Please ensure that your "
+                    "intended behavior aligns with the automatic model selection."
+                )
+        else:
+            # If auto=False, check if damped_trend has been passed by user
+            if damped_trend:
+                self.damped_trend = damped_trend
```

Comment:
this will break the `sktime` interface contract.
If you read the extender contract, attributes of self should always have the same value as the `__init__` inputs.